src/components/ebay-notice/migrator.js

```diff
@@ -0,0 +1,59 @@
+/**
+ * @description
+ * Migrates each notice to different types
+ */
+
+// Types of buttons that if they are direct children of root will be wrapped in footer
+const buttonTypes = ['ebay-button', 'ebay-menu-button'];
+
+function setAttribute(el, oldName, newName) {
```

Comment:
Looks like this function only sets the attribute if it was already there, is that intentional?

Reply:
Yes, it's renaming it from the old name to the new.